src/formatters/captureChoiceFormatter.ts

```diff
@@ -171,16 +170,19 @@ export class CaptureChoiceFormatter extends CompleteFormatter {
 			return -1;
 		}
 
-		if (fileCache.frontmatter.position)
-			return fileCache.frontmatter.position.end.line;
+		if (fileCache.frontmatter.position || fileCache.frontmatterPosition) {
+			if (fileCache.frontmatter.position) {
+				return fileCache.frontmatter.position.end.line;
+			}
+		}
```

Comment:
This change isn't actually doing anything because the `fileCache.frontmatterPosition` is discarded. @evanaze did you mean do discard `frontmatter.position` instead of `frontmatterPosition`?